tests/ltp/Makefile

```diff
@@ -20,10 +20,34 @@ ifndef FS
 FS=ext2fs
 endif
 
+# *** start fork related settings ***
+# fork is now compiled into the code all the time and is controled
+# through config. As LTP needs fork to function we now need to 
+# use the fork enabled ltp tests, and we need to use the config.json
+# to use the fork implementation
+
+#original ltp source
+# URL=https://github.com/linux-test-project/ltp
+# BRANCH=master
+
+# fork enabled
+URL=https://github.com/paulcallen/ltp
+BRANCH=paulcallen/revert-fork
+
+# Fork disabled
+#URL=https://github.com/mikbras/ltp
+#BRANCH=myst
```

Comment:
Do we still need Mike's ltp repo? If not delete these lines.

Reply:
removed